### ISSUE TITLE
Shape must be of rank 4 but is of rank 3

### ISSUE THREAD

**Issue by pradiepsingh** [jiegzhan/multi-class-text-classification-cnn-rnn#34]
Hi,

While running train.py I am getting below error:

Shape must be rank 4 but is rank 3 for 'conv-maxpool-3/concat_2' (op: 'ConcatV2') with input shapes: [?,1,300,1], [?,548,1], [?,1,300,1], [].

This is happening in text_cnn_rnn.py at 
conv = tf.nn.conv2d(emb_pad, W, strides=[1, 1, 1, 1], padding='VALID', name='conv')

Can any one support in providing some help on this.

**Comment by jiegzhan** [comment]
I just ran the train.py, it works for me. 

python3 train.py ./data/train.csv.zip ./training_config.json

This is tensorflow on my machine: tensorflow-gpu (1.3.0)

**Comment by pradiepsingh** [comment]
Thanks fir your reply,
Just want to tell you I am running it on python 3 tensorflow 1 on Windows is it the problem?
Also not able to call sys.argv[1] it is showing list index out of range for calling the upload path of data_helper.py

**Comment by pradiepsingh** [comment]
Hi Jie,
One more thing I would like to know do I need these modules to run on Apache?
If you can post some solution for Windows..

**Comment by pradiepsingh** [comment]
Hi Jie,
Thanks it is working.
